### PR TITLE
Ui sprint2

### PR DESCRIPTION
**Pull request by miguelcg88**
pull request last version UI

### CODE REVIEW

File: .idea/libraries/file4.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```
Comment (GabrielRendon) [inline .idea/libraries/file4.xml:1]: Is this file nedded? if not, please remove it.

File: .idea/modules.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (GabrielRendon) [inline .idea/modules.xml:1]: Is this file nedded? if not, please remove it.

File: .idea/uiDesigner.xml [hunk 3]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (GabrielRendon) [inline .idea/uiDesigner.xml:1]: Is this file nedded? if not, please remove it.

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (GabrielRendon) [inline .idea/vcs.xml:1]: Is this file nedded? if not, please remove it.

File: .idea/libraries/file4.xml (hunk 1 above)
Comment (miguelcg88) [reply .idea/libraries/file4.xml:1]: I tested without file and could run program, so it is needed

File: .idea/modules.xml (hunk 2 above)
Comment (miguelcg88) [reply .idea/modules.xml:1]: I tested without file and could run program, so it is needed

File: .idea/uiDesigner.xml (hunk 3 above)
Comment (miguelcg88) [reply .idea/uiDesigner.xml:1]: I tested without file and could run program, so it is needed

File: .idea/vcs.xml (hunk 4 above)
Comment (miguelcg88) [reply .idea/vcs.xml:1]: I tested without file and could run program, so it is needed

**Pull request merged**